code.swift

```diff
@@ -54,25 +54,24 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
```

Comment:
That’s actually a necessary bit of code that you removed! There is something else that can be improved on this line though...